### PR TITLE
CRPL-1489: Adds namespace & usage info to the root readme

### PR DESCRIPTION
**Pull request by amatiszcr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,83 @@
...
+> - Custom prototypes created in the reserved namespaces
+> - Changes to the public prototypes  
+> 
+> For your own work, use ***Private namespaces*** instead.
+
+
+# Private namespaces
+
+By convention, private namespaces should follow the ```#{name}``` naming convention - these will be ignored during upgrade/install.
+The installer will create a couple of these by default:
```
Comment (lvass74) [inline README.md:41]: Typo: couple -> copy

File: README.md [hunk 2]
```diff
@@ -1,11 +1,83 @@
...
+```
+#team
+#user
+#workspace
+```
+Currently, any new node created through the CRANQ IDE will be automatically placed in the ```#workspace``` namespace. 
+
+> **_How do I set/change the namespace of my node?_**
+>
+> Just rename your prototype in the CRANQ IDE, with the namespace path you with to use:
```
Comment (lvass74) [inline README.md:51]: Typo: "you with to use" -> "you wish to use"

File: README.md [hunk 3]
```diff
@@ -1,11 +1,83 @@
...
+```
+You can use this file to change the folder location of your private namespaces.
+- Use fully qualified paths for now
+- Restart CRANQ for your changes to take effect
+- You don't have to explicitly create a record here for all of your private namespaces - by default, everything is placed under ```""```, where the public ones are located
+- Changing the path for ```""``` will change the default namespace path, but it is not recommeded at the moment
+
+> **_Can I map my private namespaces to a git repository?_**
+>
+> Yes, absolutely! Just set your namespace path in this file to your repository, and you are set.
```
Comment (lvass74) [inline README.md:78]: Rather "to your locally cloned repository path"?
To avoid people putting in the git repository URL instead of the local path

**Review by lvass74 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lvass74) [reply README.md:41]: My bad, ignore the above comment!

File: README.md (hunk 2 above)
Comment (amatiszcr) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (amatiszcr) [reply README.md]: Fixed

**Pull request merged**